comp/trace/config/setup.go

```diff
@@ -232,8 +232,12 @@ func applyDatadogConfig(c *config.AgentConfig, core corecompcfg.Component) error
 		c.MaxEPS = core.GetFloat64("apm_config.max_events_per_second")
 	}
 	if core.IsSet("apm_config.max_traces_per_second") {
+		log.Warn("`apm_config.max_traces_per_second` is deprecated, please use `apm_config.target_traces_per_second` instead")
```

Comment:
Is there anything we can link to here or elsewhere that will let users know what's happening?

As a user, I might be confused about how to update my config if I don't know that `max_traces_per_second` and `target_traces_per_second` do the same thing.

Maybe pointing at the release notes or the config template is sufficient?

Reply:
As discussed offline, we will leave it as is, not to have to maintain this link (which can potentially be dead). We will sync with TEEs to give a heads up on the change, to anticipate these issues if customers raise them.